Gemfile.lock

```diff
@@ -6,8 +6,30 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
+    ast (2.4.0)
```

Comment:
Are these deps just for the build/test setup or do they also get included for users that install the gem?

Reply:
We only have dev dependencies. We only depend on the standard library at runtime.